src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -386,7 +385,7 @@ def link_content(text, css_class, icon)
     if css_class && css_class.include?('nav-link')
       capture do
         concat(tag.i('', class: "fas #{icon}"))
-        concat(tag.div(text, class: 'small'))
+        concat(tag.div(text, class: feature_enabled?(:responsive_ux) ? '' : 'small'))
```

Comment:
[To fix the font size issue found by @krauselukas](https://github.com/openSUSE/open-build-service/pull/10007#issuecomment-673463363)